### PR TITLE
Add plugins support

### LINKED ISSUE #11: Ability to add plugins
Hello! 
It would be great to add ability to add plugins for storages with autoconfigure `\League\Flysystem\PluginInterface` interface

### PR DESCRIPTION
**Pull request by BoShurik**
Fixes #11

### CODE REVIEW

**Comment by tgalopin** [comment]
Thanks very much, I'll have a look as soon as possible :) !

File: tests/Kernel/config/services.yaml [hunk 1]
```diff
@@ -21,3 +21,6 @@ services:
     flysystem.test.fs_sftp: { alias: 'fs_sftp' }
     flysystem.test.fs_webdav: { alias: 'fs_webdav' }
     flysystem.test.fs_zip: { alias: 'fs_zip' }
+
+    Tests\League\FlysystemBundle\Plugin\DummyPlugin:
+        autoconfigure: true
```
Comment (tgalopin) [inline tests/Kernel/config/services.yaml]: Missing EOF line

Comment (BoShurik) [reply tests/Kernel/config/services.yaml]: @tgalopin done

**Comment by tgalopin** [comment]
Thanks @BoShurik !

**Pull request merged**